pages/common/php-yii.md

```diff
@@ -0,0 +1,15 @@
+# php yii
+
+> Yii Framework's command line interface.
+
+- Display a list of all available commands:
+
+`php yii {{help}}`
```

Comment:
Wouldn't this execute a file called `yii` in the current directory? I assume that a dash `-` is missing here, or the `php` bit is extraneous, or something else?

Not familiar with the _Yii Framework_, so I could be wrong.

Reply:
That is correct. Mentioned more in the [php artisan PR](https://github.com/tldr-pages/tldr/pull/2346#discussion_r219605667)

Reply:
Cool, thanks